airbyte-integrations/connectors/source-instagram/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "3.0.20"
+version = "3.1.0"
```

Comment:
It is missing to bump the version in the metadata.yaml file too. Can you do that?

Reply:
Done